lib/internal/test/unicode.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+// This module exists entirely for regression testing purposes.
+// See `test/parallel/test-internal-unicode.js`.
```

Comment:
maybe worth putting unicode in the comment, it looks like some tools care about this.